ui/src/auth/Authorized.js

```diff
@@ -51,10 +50,12 @@ const Authorized = ({
   // React.isValidElement guards against multiple children wrapped by Authorized
   const firstChild = React.isValidElement(children) ? children : children[0]
 
-  const meRole = getMeRole(me)
+  if (!isUsingAuth) {
+    return replaceWithWhenNotUsingAuth || firstChild
+  }
 
-  if (!isUsingAuth || isUserAuthorized(meRole, requiredRole)) {
-    return firstChild
+  if (isUserAuthorized(meRole, requiredRole)) {
+    return replaceWith || firstChild
```

Comment:
We discussed in Slack that this degrades the clarity of the HOC's functionality

Reply:
conclusions:

Reply:
takeaways: 1. replace `replaceWith` with `replaceWithIfAuthorized` and `replaceWithIfNotAuthorized` and amend the above logic accordingly. then, correct ```<Authorized
                requiredRole={SUPERADMIN_ROLE}
                replaceWith={
                  <input
                    type="text"
                    value={userOrganization.name}
                    disabled={true}
                    className="form-control input-sm disabled"
                  />
                }
              >``` to use that new API and replace all other instances of `replaceWith` with the correct new API

Reply:
fixed -- great catch